Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml

```diff
@@ -396,16 +397,16 @@
   - type: Clothing
     sprite: Clothing/Uniforms/Jumpsuit/janitor.rsi
 
-- type: entity
-  parent: ClothingUniformBase
-  id: ClothingUniformJumpsuitKimono
-  name: kimono
-  description: traditional chinese clothing
-  components:
-  - type: Sprite
-    sprite: Clothing/Uniforms/Jumpsuit/kimono.rsi
-  - type: Clothing
-    sprite: Clothing/Uniforms/Jumpsuit/kimono.rsi
+# - type: entity
+#   parent: ClothingUniformBase
+#   id: ClothingUniformJumpsuitKimono
+#   name: kimono
+#   description: traditional chinese clothing
+#   components:
+#   - type: Sprite
+#     sprite: Clothing/Uniforms/Jumpsuit/kimono.rsi
+#   - type: Clothing
+#     sprite: Clothing/Uniforms/Jumpsuit/kimono.rsi
```

Comment:
```suggestion
```

If you are moving these to a different location, you do not need to leave the originals commented.